packages/binding-modbus/src/modbus-client.ts

```diff
@@ -110,6 +112,22 @@ export default class ModbusClient implements ProtocolClient {
     return operation.execute()
   }
 
+  private validateEndianness(form: ModbusForm) : ModbusEndianness {
+    let endianness = ModbusEndianness.BIG_ENDIAN
+    if(form.contentType) {
+      const contentValues : string[] = form.contentType.split(';') ?? []
+      // Check endian-ness
+      const byteSeq = contentValues.find(value => /^byteSeq=/.test(value))
+      if(byteSeq) {
+        const guessEndianness = byteSeq.split('=')[1] as ModbusEndianness
+        if(guessEndianness) {
```

Comment:
What if guessEndianness is not among the admitted values? the function seems to just extract the right value, but it does not a proper validation

Reply:
Should I throw an error or just discard the parameter (falling back to Big Endian)?